src/plugins/intel_npu/src/utils/src/zero/CMakeLists.txt

```diff
@@ -15,9 +15,12 @@ ov_add_clang_format_target(${TARGET_NAME}_clang FOR_TARGETS ${TARGET_NAME})
 add_library(openvino::npu_zero_utils ALIAS ${TARGET_NAME})
 set_target_properties(${TARGET_NAME} PROPERTIES EXPORT_NAME npu_zero_utils)
 
+add_library(level-zero-headers INTERFACE)
+set_property(TARGET level-zero-headers APPEND PROPERTY INTERFACE_INCLUDE_DIRECTORIES $<BUILD_INTERFACE:${OpenVINO_SOURCE_DIR}/thirdparty/level_zero/level-zero/include>)
+
 target_include_directories(${TARGET_NAME}
   PUBLIC
-    $<TARGET_PROPERTY:LevelZero::LevelZero,INTERFACE_INCLUDE_DIRECTORIES>
+    $<TARGET_PROPERTY:level-zero-headers,INTERFACE_INCLUDE_DIRECTORIES>
```

Comment:
One minor comment: Can we add an alias (LevelZero::Headers or something like this) like the one below to keep the same design?